### PR TITLE
Doc 12711 CBL 3-1-10 Maintenance Release Documentation.

### PR DESCRIPTION
**Pull request by ElliotFrancisHunter**
This is a maintenance release for CBL 3.1.10.
Also some flyby fixes

- Tidied formatting of download version links.
- Added downloads to C quickstart page

 Home: https://preview.docs-test.couchbase.com/maint3110/home/index.html 

## Preview Credentials: https://couchbasecloud.atlassian.net/wiki/spaces/DOCS/pages/1971224949/Docs+Team+demo+site+passwords

Changed Pages
-------------------

- CBL-C Downloads : https://preview.docs-test.couchbase.com/maint3110/couchbase-lite/current/c/gs-downloads.html
- C Quckstart: https://preview.docs-test.couchbase.com/maint3110/couchbase-lite/current/c/quickstart.html
- Android Release Notes: https://preview.docs-test.couchbase.com/maint3110/couchbase-lite/current/android/releasenotes.html#maint-3-1-10
- C Release Notes: https://preview.docs-test.couchbase.com/maint3110/couchbase-lite/current/c/releasenotes.html#maint-3-1-10
- .NET Release Notes: https://preview.docs-test.couchbase.com/maint3110/couchbase-lite/current/csharp/releasenotes.html#maint-3-1-10
- Java Release Notes: https://preview.docs-test.couchbase.com/maint3110/couchbase-lite/current/java/releasenotes.html#maint-3-1-10
- Objective-C Release Notes: https://preview.docs-test.couchbase.com/maint3110/couchbase-lite/current/objc/releasenotes.html#maint-3-1-10
- Swift Release Notes: https://preview.docs-test.couchbase.com/maint3110/couchbase-lite/current/swift/releasenotes.html#maint-3-1-10

----------------------
Pages still needing changes/updates
-----------------------

....

-----------------------

### CODE REVIEW

File: modules/c/partials/release-notes/couchbase-mobile-c-release-note.3.1.10.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+=== Enhancements
+
+None for this release
+
+=== Issues and Resolutions
+// Lite Core Start
+* https://issues.couchbase.com/browse/CBL-6282[CBL-6282 -- Replicator starts slowly for big databases]
+
+* https://issues.couchbase.com/browse/CBL-6377[CBL-6377 -- Fixed crash when calling onWebSocketGotTLSCertificate callback after the connection is closed]
+// Lite Core End
```
Comment (osfameron) [inline modules/c/partials/release-notes/couchbase-mobile-c-release-note.3.1.10.adoc:15]: These are comments you put in as standard now, to differentiate between the tickets shared across all mobile APIs and the API-specific ones?

File: modules/csharp/partials/release-notes/couchbase-mobile-csharp-release-note.3.1.10.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+=== Enhancements
+
+None for this release
+
+=== Issues and Resolutions
+// Lite Core Start
+* https://issues.couchbase.com/browse/CBL-6282[CBL-6282 -- Replicator starts slowly for big databases]
+
+* https://issues.couchbase.com/browse/CBL-6377[CBL-6377 -- Fixed crash when calling onWebSocketGotTLSCertificate callback after the connection is closed]
+// Lite Core End
```
Comment (osfameron) [inline modules/csharp/partials/release-notes/couchbase-mobile-csharp-release-note.3.1.10.adoc:13]: Aha, I see, yes makes more sense for this one that has additional non-LiteCore tickets 👍

**Review by osfameron — APPROVED** [review]
Looks good to me from overview of diff (I've not spent time on the content or the preview build, let me know if you need further review from me)

File: modules/c/partials/release-notes/couchbase-mobile-c-release-note.3.1.10.adoc (hunk 1 above)
Comment (ElliotFrancisHunter) [reply modules/c/partials/release-notes/couchbase-mobile-c-release-note.3.1.10.adoc:15]: Yes, makes it easier for me to quickly put these together with less risk of making errors. Just define the core ones first, copy them across and then add platform specific tickets afterwards

**Review by bmeike — APPROVED** [review]
Java and Android look good to me, once those URLs get fixed.

File: modules/android/partials/release-notes/couchbase-mobile-android-release-note.3.1.10.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+=== Enhancements
+
+None for this release
+
+=== Issues and Resolutions
+// Lite Core Start
+* https://issues.couchbase.com/browse/CBL-6282[CBL-6282 -- Replicator starts slowly for big databases]
+
+* https://issues.couchbase.com/browse/CBL-6377[CBL-6377 -- Fixed crash when calling onWebSocketGotTLSCertificate callback after the connection is closed]
```
Comment (pasin) [inline modules/android/partials/release-notes/couchbase-mobile-android-release-note.3.1.10.adoc]: This is CBL-C issue only. I have made the ticket private and create another one, only for C.

Comment (ElliotFrancisHunter) [reply modules/android/partials/release-notes/couchbase-mobile-android-release-note.3.1.10.adoc]: Fixed, thank you

**Pull request merged**